src/components/GameItem.js

```diff
@@ -4,23 +4,56 @@ import ItemIcons from '../ItemIcons.js';
 import PropTypes from 'prop-types';
 
 class GameItem extends Component {
+  constructor(props) {
+    super(props);
 
-
-  render() {
-    const itemStyle = {
-      bottom: `${this.props.height}px`, // use props.height to offset from the bottom of screen
-      zIndex: this.props.layer, // use props.layer to set z-index, so we display ontop of background
+    this.state = {
+      spotted: false
     };
 
-    // Update this to select the correct icon for each item
-    const icon = ItemIcons.rock;
+  }
+
+  onItemClick = () => {
+    this.setState({
+      spotted: true
+    });
 
-    return (
-      <div className="game-item" style={itemStyle}>
-        <img src={icon} alt="Item" className="icon-item"></img>
-      </div>
-    );
+    if (this.props.type === 'litter') {
```

Comment:
I suggest also making sure that the item has not been clicked on previously.  I was able to run up my score by mashing the mouse on a single piece of litter.